pahfit/features/features.py

```diff
@@ -82,7 +82,7 @@ def value_bounds(val, bounds):
     if val is None:
         val = np.ma.masked
     if not bounds:
-        return (val,) + 2 * (np.ma.masked,)  # Fixed
+        return (val,) + 2 * (np.nan,)  # (val,nan,nan) indicates fixed
```

Comment:
You said that a single np.nan would represent infinity, but both np.nan represents fixed. How are [-inf, +inf] bounds represented then? I assume -np.inf, +np.inf? 

Am I correct to interpret this as?: we should never have a single np.nan in there, but if it happens, interpret it as np.inf as a failsafe

Reply:
The use case I'm seeing here: its set to (val,nan,nan), and the user only changes one bound e.g. bla['max'] = 1.0. I think this will happen relatively commonly.

Reply:
In fact your example below (just altering the `'max'`) does exactly this, so it would be natural!